data/data/alibabacloud/slb_attach_controlplane/variables.tf

```
@@ -0,0 +1,14 @@
+variable "slb_ids" {
+  type        = list(string)
+  description = "The load balancer IDs of the bootstrap ECS."
```

This description is not correct.
```suggestion
  description = "The IDs of the load balancers to which to attach the bootstrap and control plane VMs."
```